tests/npm-app/src/types/ccf.ts

```diff
@@ -263,4 +264,14 @@ export class TypedKVMap<K, V> {
     delete(key: K): boolean {
         return this.kv.delete(this.kt.encode(key));
     }
+    forEach(callback: (value: V, key: K, table: TypedKVMap<K, V>) => void) : void {
+        let kt = this.kt;
+        let vt = this.vt;
+        let typed_table = this;
```

Comment:
Camel case please :)

Reply:
@letmaik @eddyashton unless there's a strong objection, I will set up https://prettier.io/, because it covers everything that's currently not (JS/TS, YAML, Markdown), it requires little config, and it will solve this kind of problem forever.